schema/config_blobset_blob.json

```diff
@@ -8,29 +8,20 @@
     "phase": {
       "$ref": "file:common.json#/definitions/phase"
     },
-    "content_type": {
-      "type": "string"
-    },
-    "base64": {
-      "type": "string",
-      "pattern": "^[0-9A-Za-z=+/]{4,}$"
-    },
     "url": {
-      "type": "string",
-      "format": "uri"
+      "type": "string"
     },
     "sha256": {
       "type": "string",
       "description": "Expected hash of the retrieved resource",
       "pattern": "^[0-9a-f]{64}$"
     },
-    "nonce": {
+    "generation": {
```

Comment:
To be honest I'm still not a fan of this naming. The best case for it is that we already have it in "discovery", so it's consistent to have a field with this name that is a date-time value.  The other fields in the schema tell you what they are by their names, and you could reasonably understand the value. This naming gives you no such hint. "Timestamp" would be more clear. WDYT?

Reply:
but it's not just a timestamp -- the term "generation" has the semantics of grouping several things together (e.g. multiple state changes that are all part of the same overall operation), which is different than the basic timestamp of each individual state change. It also is a timestamp in that it can be used to generally sort things into rough categorization of when things happen, which is useful for debugging in a distributed system (which is not something then you can easily do with a random nonce).

Reply:
As we discussed in GVC, I agree the value may become a date-time format. I disagree it should be generation. 

The maximal improvement would be to name it and make it a timestamp, because the value will give you the human debug utility you request, and the name neither overly enhances it (since we aren't going to enforce any kind of grouping or data consistency across the "generation" fields) nor under-describes it (it's clear the field is a date/time).

If you'd like to proceed go ahead but I think the more general "timestamp" is optimal.

Reply:
> and the name neither overly enhances it (since we aren't going to enforce any kind of grouping or data consistency across the "generation" fields)

But it does -- the "generation" will remain consistent across multiple phases of an operation. So, if there's "apply" and "final" they would have the same _generation_ but occur at different individual times (_timestamp_). This is why I don't want to call it "timestamp" since it has a semantically different meaning.

Reply:
I don't think timestamp needs to change just because it's called timestamp, but I do think generation could also be "3rd" or (int)0 or "nonce". My suggestion is generation_timestamp since it retains what I understand to be both our highest level concerns.